docs/literate/src/files/getting_started.jl

```diff
@@ -0,0 +1,288 @@
+#src # Getting started
+
+# Trixi.jl is a numerical simulation framework for conservation laws and
+# is written in the [Julia programming language](https://julialang.org/).
+# This tutorial is intended for beginners in Julia and Trixi.jl.
+# After reading it, you will know how to install Julia and Trixi.jl on your computer,
+# and you will be able to download setup files from our GitHub repository, modify them,
+# and run simulations.
+
+# ## Julia installation
```

Comment:
I have tried `juliaup` on Windows:
```
winget install julia -s msstore
```
It works for me only once, then I tried to uninstall and install `julia` again. Unfortunately I'm experiencing now every time a `server error`. I found many reports about this failure. Moreover there exists [way around](https://discourse.julialang.org/t/looking-for-feedback-from-users-that-currently-cant-install-juliaup-from-the-windows-store/85507) using `winget` command. But I'm not sure, that it will be long supported. So it may be better to leave that longer, but less error prone method?
I also tried `juliaup` on Xubuntu using VirtualBox:
```
curl -fsSL https://install.julialang.org | sh
```
Installation fails, but it may be caused by VirtualBox specific.